hnn_core/gui/gui.py

```diff
@@ -1378,6 +1441,73 @@ def run_button_clicked(widget_simulation_name, log_out, drive_widgets,
                                           plot_type, {}, "plot")
 
 
+def _serialize_simulation(log_out, sim_data, simulation_list_widget):
+    # Only download if there is there is at least one simulation
+    sim_name = simulation_list_widget.value
+
+    with log_out:
+        logger.info(f"Saving {sim_name}.txt")
+        return serialize_simulation(sim_data, sim_name)
+
+
+def serialize_simulation(simulations_data, simulation_name):
+    """
+        Serializes the simulation data for a given simulation name
+        into either a single CSV file
+        or a ZIP file containing multiple CSVs, depending on the number
+        of trials in the simulation.
+        Parameters:
+        - simulation_name (str): The name of the simulation to serialize.
```

Comment:
Can you fix the docstring to conform to the format used in the repository? You can look at any of the public functions to see examples

Reply:
Done!